README.md

```diff
@@ -0,0 +1,271 @@
+
+### **Development**
+
+[![coveralls](https://img.shields.io/coveralls/github/biocommons/hgvs.svg)](https://coveralls.io/github/biocommons/hgvs) [![issues](https://img.shields.io/github/issues-raw/biocommons/hgvs.svg)](https://github.com/biocommons/hgvs/issues)
+[![GitHub Open Pull Requests](https://img.shields.io/github/issues-pr/biocommons/hgvs.svg)](https://github.com/biocommons/hgvs/pull/) [![GitHub license](https://img.shields.io/github/contributors/biocommons/hgvs.svg)](https://github.com/biocommons/hgvs/graphs/contributors/) [![GitHub stars](https://img.shields.io/github/stars/biocommons/hgvs.svg?style=social&label=Stars)](https://github.com/biocommons/hgvs/stargazers) [![GitHub forks](https://img.shields.io/github/forks/biocommons/hgvs.svg?style=social&label=Forks)](https://github.com/biocommons/hgvs/network)
+
+## **Features**
+
+-   Parsing is based on formal grammar.
+-   An easy-to-use object model that represents most variant types
+    (SNVs, indels, dups, inverstions, etc) and concepts (intronic
+    offsets, uncertain positions, intervals)
+-   A variant normalizer that rewrites variants in canoncial forms and
+    substitutes reference sequences (if reference and transcript
+    sequences differ)
+-   Formatters that generate HGVS strings from internal representations
+-   Tools to map variants between genome, transcript, and protein
+    sequences
+-   Reliable handling of regions genome-transcript discrepancies
+-   Pluggable data providers support alternative sources of transcript
+    mapping data
+-   Extensive automated tests, including those for all variant types and
+    \"problematic\" transcripts
+-   Easily installed using remote data sources. Installation with local
+    data sources is straightforward and completely obviates network
+    access
+
+## **Important Notes**
+
+-   **You are encouraged to** [browse
+    issues](https://github.com/biocommons/hgvs/issues). All known issues
+    are listed there. Please report any issues you find.
+-   **Use a pip package specification to stay within minor releases.**
+    For example, `hgvs>=1.5,<1.6`. hgvs uses [Semantic
+    Versioning](http://semver.org/).
+
+## **Installation**
+
+**Important:** For more detailed installation and configuration
+instructions, see the [HGVS readthedocs](https://hgvs.readthedocs.io/)
+
+### Prerequisites
+
+    libpq
+    python3
+    postgresql
+
+#### **Examples for installation:**
+
+#### MacOS :
+
+    brew install libpq
+    brew install python3
+    brew install postgresql@14
+
+#### Ubuntu :
+
+    sudo apt install gcc libpq-dev python3-dev
+
+### Installation Steps
+
+By default, hgvs uses remote data sources, which makes
+installation easy.
+
+1. Create a virtual environment using your preferrred method.
+
+    **Examples:**
+
+    #### MacOS :
+
+        virtualenv venv
```

Comment:
`python3 -m venv venv` works with MacOS. I think it's fine to revert back to the original README and stick with venv.

Reply:
done!